images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -47,14 +50,20 @@ function Set-JavaPath {
     }
 }
 
-function Install-JavaFromAdoptOpenJDK {
+function Install-JavaJDK {
     param(
         [string] $JDKVersion,
-        [string] $Architecture = "x64"
+        [string] $Architecture = "x64",
+        [string] $FullVendorName
     )
 
-    # Get Java version from adopt openjdk api
-    $assetUrl = Invoke-RestMethod -Uri "https://api.adoptopenjdk.net/v3/assets/latest/${JDKVersion}/hotspot"
+    # Get Java version from api
+    if ($FullVendorName -eq "Termurin-Hotspot") {
+        $assetUrl = Invoke-RestMethod -Uri "https://api.adoptium.net/v3/assets/latest/${JDKVersion}/hotspot"
+    }
```

Comment:
Please use consistent styling.
If open bracket on the same line with else - let's put close bracket on the same line.

```
if () {
  
} else {

}
```

Reply:
Makes sense